data/sql/updates/pending_db_world/rev_1643833630496479400.sql

```diff
@@ -0,0 +1,17 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1643833630496479400');
+
+SET @NPC_ENTRY := 235; /* Salma Saldean */
+SET @GOSSIP_MENU_ID := 61027;
+SET @NPC_TEXT_ID := 50033;
+
+UPDATE `creature_template` SET `gossip_menu_id` = @GOSSIP_MENU_ID WHERE `entry` = @NPC_ENTRY;
+UPDATE `creature_template` SET `npcflag` = 3 WHERE `entry` = @NPC_ENTRY;
```

Comment:
```suggestion
UPDATE `creature_template` SET `npcflag` = `npcflag`|1|2 WHERE `entry` = @NPC_ENTRY;
```
https://www.azerothcore.org/wiki/sql-standards#flags-amp-bits

Reply:
Thank you! Sorry to have missed this SQL standard. I will fix for my future PRs.